lejos-mf-pc

```diff
@@ -1 +1 @@
-Subproject commit 3e4282d467ef284ffc03c5d872fb238d07a3e608
+Subproject commit 97883cbacf0b9fe8aad866fd2241917b87cda6c0
```

Comment:
It was indeed a mistake, well spotted! This should not be here, the version is not good... Will update it.